cl/assets/static-global/js/toggle_settings.js

```diff
@@ -10,64 +10,47 @@ function toggleSettings(e) {
     alert_text = button.find('.alert_btn_txt'),
     url = button.attr('href');
 
-  button_icon
-    .removeClass()
-    .addClass('fa fa-spinner fa-pulse');
+  button_icon.removeClass().addClass('fa fa-spinner fa-pulse');
   $.ajax({
     method: 'POST',
     url: url,
-    data: {'id': id},
+    data: { id: id },
     success: function (data) {
-      $('.bootstrap-growl').alert("close");
-      $.bootstrapGrowl(
-        data,
-        {
-          type: "success",
-          align: "center",
-          width: "auto",
-          delay: 2000,
-          allow_dismiss: false,
-          offset: {from: 'top', amount: 80}
-        }
-      );
+      $('.bootstrap-growl').alert('close');
+      $.bootstrapGrowl(data, {
+        type: 'success',
+        align: 'center',
+        width: 'auto',
+        delay: 2000,
+        allow_dismiss: false,
+        offset: { from: 'top', amount: 80 },
+      });
       if (/disabled/g.exec(data)) {
-        button_icon
-          .removeClass()
-          .addClass("fa " + button_enable_class);
-        button
-          .removeClass("btn-danger")
-          .addClass("btn-success");
+        button_icon.removeClass().addClass('fa ' + button_enable_class);
+        button.removeClass('btn-danger').addClass('btn-success');
         alert_text.text(enable_message);
 
-        button.next()[0].className = "btn btn-success dropdown-toggle";
-
+        button.next('.dropdown-toggle')[0].className = 'btn btn-success dropdown-toggle';
```

Comment:
```suggestion
        if (button.next('.dropdown-toggle')[0]){
          button.next('.dropdown-toggle')[0].className = 'btn btn-success dropdown-toggle';
        }
```
Testing this fix seems that it's necessary to check if the `dropdown-toggle` object element exists before otherwise a `Cannot set properties of undefined` error arises and when toggling the button its action text doesn't change when the button doesn't have a `drop down`.

![Screen Recording 2022-09-26 at 17 47 14](https://user-images.githubusercontent.com/486004/192394919-8427581e-c333-4869-ae94-71de0fd385ee.gif)

![Screen Recording 2022-09-26 at 17 47 31](https://user-images.githubusercontent.com/486004/192394930-50bea521-be03-45a8-bd51-4731e2c8006a.gif)

After applying the suggestion seems that it works well and also the `Learn more...` bug it's solved.

![Screen Recording 2022-09-26 at 17 48 35](https://user-images.githubusercontent.com/486004/192395185-e89dee6c-4433-4ada-b092-979cf12c672f.gif)

![Screen Recording 2022-09-26 at 17 48 18](https://user-images.githubusercontent.com/486004/192395184-73346fcf-5e79-46a5-abff-1e2b99c7d364.gif)